incubator-tools/clean_amount_data/clean_amount_data.ipynb

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
<p>Change this description to be more clear what the notebook does. It just seems like it does some reformatting on certain money values.</p><p> </p><p>I'm not actually sure this is needed if Normalized Values are supported.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
yes normalized values supports, but we are not changing any other entity-class-attribute otherthan entity.mention_text(irrespective of normalized-values, `mention_text attr contains unexpected spaces, commas & periods`).This tool performs `post-processing on entity.mention_text` and makes it business redable string-format.